src/django/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN set -ex \
     && deps=" \
     gdal-bin \
     gettext \
-    postgresql-client-13 \
+    postgresql-client-15 \
```

Comment:
Is this version correct?

Reply:
Yeah...
I was researching this yesterday and on Monday, and I found the following:
The [postgresql-client-16](https://packages.debian.org/search?keywords=postgresql-client-16&searchon=names&suite=all&section=all) package isn't available for the latest stable version of Debian, which is [Bookworm](https://www.debian.org/releases/), used in the Django container (see line 1 in the Dockerfile). The latest stable version available is [postgresql-client-15](https://packages.debian.org/search?keywords=postgresql-client-15&searchon=names&suite=all&section=all), which is why I used it. Generally, there shouldn't be any issues, as PostgreSQL versions are usually compatible with each other. Testing the Django app locally confirms that it works as expected.

Reply:
Thank you!